bodhi/tests/server/functional/test_admin.py

```diff
@@ -0,0 +1,44 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import copy
+from webtest import TestApp
+
+from bodhi.server import main
+from bodhi.server.models import (User, Group)
+import bodhi.tests.server.functional.base
+
+
+class TestAdminView(bodhi.tests.server.functional.base.BaseWSGICase):
+
+    def test_admin(self):
+        """Test that authticated user can see the Admin panel"""
+        res = self.app.get('/admin/')
+        body = res.json_body
+        self.assertEquals(body['principals'][0], 'system.Everyone')
+        self.assertEquals(body['principals'][1], 'system.Authenticated')
+        self.assertEquals(body['principals'][2], 'guest')
+        self.assertEquals(body['user'], 'guest')
```

Comment:
yeah, not really sure of the merit of it, but i added the tests anyway, just in case we make it more useful in the future...